src/MagnumPlugins/BasisImporter/BasisImporter.conf

```diff
@@ -0,0 +1,13 @@
+provides=BasisImporterEtc1
+provides=BasisImporterEtc2
+provides=BasisImporterBc1
+provides=BasisImporterBc3
+provides=BasisImporterBc4
+provides=BasisImporterBc5
+provides=BasisImporterBc7M6OpaqueOnly
+provides=BasisImporterPvrtc1_4OpaqueOnly
+
+# [configuration]
+[configuration]
+format=Etc2
```

Comment:
I assume this should be `Format` analog to Assimp configuration values? :thinking:

Reply:
no, those are uppercase only because the Assimp internal enums are named that way too (and so I have the mapping easier)